### PR TITLE
Feature/derp 253 view rich text component

### PR DESCRIPTION
**Pull request by philwolstenholme**
This PR is MASSIVE, mostly because I added ESLint, Stylelint, and Prettier to the theme and ran it on the existing codebase. I should really have done this in a separate PR but I was already working across four different repos for the Rich text work and didn't fancy also having a new branch to deal with too. 

This PR:
- Adds frontend linting tools and connects them to a npm script and to Grum
- Adds styling for rich text
- Adds styling for tables
- Adds styling for custom scrollbars
- Adds a Storybook story demonstrating the spacing scale used on the site
- Adds a presenter Twig template to map the Rich text paragraph's rich text field to our Rich text component

### CODE REVIEW

**Review by joz-cti — APPROVED** [review]

**Comment by joz-cti** [comment]
@philwolstenholme PHPCS is failing

File: components/01-atoms/table/_table.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+  thead {
+    color: #111827;
+    font-weight: $t-font-weight-bold;
+  }
+
+  th {
+    padding: $table-inner-spacing;
+    vertical-align: top;
+    color: #fff;
```
Comment (lukewhitmore) [inline components/01-atoms/table/_table.scss:41]: Feels like this should be replaced with $t-colors-white?

Thinking about circumstances where alternate palettes might be desireable (i.e. high contrast. Makes potential proposition a lot less hassle for little effort maybe?

File: components/01-atoms/table/_table.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    border-color: rgba(255, 255, 255, 0.1);
+    border-right-width: 1px;
+    border-bottom-width: 1px;
+    background-color: #353d42;
```
Comment (lukewhitmore) [inline components/01-atoms/table/_table.scss:45]: Should this also be a token?

File: components/01-atoms/table/_table.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    & *:first-child {
+      border-top-left-radius: $t-border-radius-6;
+    }
+
+    & *:last-child {
+      border-top-right-radius: $t-border-radius-6;
+    }
+
+    td {
+      border-top-width: 1px;
```
Comment (lukewhitmore) [inline components/01-atoms/table/_table.scss:82]: I wonder whether there's also utility in providing 1px values for borders as tokens? Again for alternate schemes (i.e. high vis)

File: components/01-atoms/table/example-markup.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
+const scrollableTableWrapper = require('./scrollable-table.twig');
+
+const tableWithThead = `<table>
```
Comment (lukewhitmore) [inline components/01-atoms/table/example-markup.js:3]: Is it worth adding some default / standard attributes here to show how the styling reacts to those?

**Review by lukewhitmore — COMMENTED** [review]
Looks good .. added a few questions / comments; but up to you if it's worth implementing?!

**Review by lukewhitmore — APPROVED** [review]

File: components/01-atoms/table/_table.scss (hunk 1 above)
Comment (philwolstenholme) [reply components/01-atoms/table/_table.scss:41]: Ah yes, there were quite a few things I've missed out on tokenising in this file - should all be sorted now

**Review by joz-cti — APPROVED** [review]

**Pull request merged**